### DISCUSSION TITLE
What do end users download now to be able to install AIR desktop apps? I used to direct them to http://get.adobe.com/air

### DISCUSSION THREAD

**Discussion by SBDBSoftware** [airsdk/Adobe-Runtime-Support#593] [Q&A]
## Problem Description

Describe your problem in detail. Include the following information:

* Which AIR SDK did you use? Hint: always try the latest version from [Adobe Labs](http://labs.adobe.com/downloads/air.html).
* Describe the affected environment: Flash vs. AIR, Android vs. iOS, all devices vs. just specific models.
* Can you reproduce it yourself, or did you get reports from other users? If so, how many users are affected (percentage)?
* If applicable, link to the Starling/Feathers issues that brought you here.

Furthermore, **always** add a link to one or more analog [Adobe Tracker](https://tracker.adobe.com) issues. This repository is not a replacement for Adobe's bug tracker!

## Steps to Reproduce

Tell us how this issue can be reproduced. Seriously: if you want this issue to be fixed, this is the part you **really** need to get nailed.
Give us a minimal example:

* As little code as possible. Get to the root of the problem!
* As few dependencies as possible. Did the problem pop up with Feathers? Try to reproduce it with Starling only.
* Remove any ANEs (except if they are vital for the problem).
* Include all necessary files (e.g. textures).

## Known Workarounds

Just in case others run into this – do you happen to know of a workaround? Otherwise, just remove this section altogether.

**Comment by ajwfrost** [comment]
Our recommendation for folk would be to use a bundled runtime in their apps, but we are looking to provide a shared runtime still, for end users to download. Currently just trying to sort out the legal pieces needed for this, we have installers pretty much ready although facing a few issues still on Big Sur..

We should have things resolved and uploaded within the next two weeks I believe.

**Comment by SBDBSoftware** [comment]
Thank you for your reply Andrew. I can imagine the job you're doing is HUUUUUGE.

**Comment by couchwc** [comment]
For anyone that was following this thread (like me) the runtime installers are available [here](https://airsdk.harman.com/runtime).

**Reply by SBDBSoftware** [reply]
Thank you Cody - very much appreciated. Is that the link that future versions of the runtime will be available on ie if end users download that link, they'll always end up with the latest version of AIR? Thanks.

**Reply by ajwfrost** [reply]
Yes that's where we'll put the latest version each time we update it. There is some code in the runtime there that will check for updates on our servers, although we've not put anything like that in place yet..
